RadixWallet/Core/DesignSystem/Components/Thumbnails.swift

```diff
@@ -348,5 +381,5 @@ extension URL {
 		return false
 	}
 
-	private static let vectorImageTypes: [String] = ["svg", "pdf"]
+	private static let unsupportedVectorImageTypes: [String] = ["pdf"]
```

Comment:
do is there any other type of vector images asides `pdf` (and the now supported `svg`)? Otherwise we can just remove the for loop and check if it has `.pdf` suffix.

Reply:
I think you're right. I'll remove the for loop and add an `isPDF` check instead.

Reply:
Addressed feedback [here](https://github.com/radixdlt/babylon-wallet-ios/pull/1338/commits/9bd93c181bcdd8077965533eaba4c849b447e99d). Defined a new method `isVectorImage(type:)` instead.

Reply:
much better!